src/reporting-config/__tests__/reporting-config-parsers.test.ts

```diff
@@ -0,0 +1,94 @@
+import { ReportingConfigApiResponse } from '../../api';
```

Comment:
This is a rename. (and I pull the fake creating function in here because it's specific for this test).